.github/workflows/deploy.yml

```
@@ -0,0 +1,16 @@
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: superfly/flyctl-actions/setup-flyctl@master
+      - run: flyctl deploy --remote-only
+        env:
+          FLY_API_TOKEN: ${{ secrets.FLY_API_TOKEN }} 
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)